### ISSUE TITLE
releaseWizard improvements from the Solr 9.0 release [LUCENE-10365]

### ISSUE THREAD

**Issue by asfimport** [apache/lucene#11401]
Being RM for Solr 9.0 release, we benefited hugely from the build- and gradle related changes that Lucene made. But we also have some improvements on the Solr side that we source back to Lucene in this issue.

---
Migrated from [LUCENE-10365](https://issues.apache.org/jira/browse/LUCENE-10365) by Jan Høydahl (@janhoy), updated Jan 25 2022
Pull requests: https://github.com/apache/lucene/pull/591

**Comment by janhoy** [comment]
@mocobeta This issue references https://github.com/apache/lucene/pull/591 but I cannot find any back-link over in the PR that points back to this issue. I'm sure that this comment will add a ref to the PR, but I'd expect something from the 23rd of August? We should probably adapt a workflow where we include "Fixes #xxx" in the commit-message of PRs.

**Comment by mocobeta** [comment]
> I cannot find any back-link over in the PR that points back to this issue.

Yes because the issue migration Jira to GitHub didn't touch any existing PRs.

> We should probably adapt a workflow where we include "Fixes #xxx" in the commit-message of PRs.

I noticed some people already use "Fixes #xxx" or "Closes #xxx" tips. I'm not so sure if we should/can make people obey any extra rules, but please feel free to propose/proceed with it.

**Comment by janhoy** [comment]
> Yes because the issue migration Jira to GitHub didn't touch any existing PRs.

The MR was not changed, but in the 9 jan comment the asfimport bot reference the linked PR, without it being reflected on the PR. But when I mentioned the PR manually we got the backlink in the PR itself.

> I'm not so sure if we should/can make people obey any extra rules

These would not be rules. If folks want to close issues manually that's ok, just as we did with JIRA. But some devs may not be familiar with the commit-msg syntax to do it automatically. A hint in docs could point them in the right direction.
